module/enrichers.mjs

```diff
@@ -570,7 +570,7 @@ async function enrichItem(config, label, options) {
   if ( givenItem.startsWith(".") ) {
     try {
       foundItem = await fromUuid(givenItem, { relative: options.relativeTo });
-    } catch { return null; }
+    } catch(err) { return null; }
```

Comment:
Is the linter complaining? If we don't use the `err` in the catch block then we don't need to declare it.

Reply:
Yeah, it complains about spacing because of how we space our catch blocks. Seems like the linter just isn't smart about it.